docs/readthedocs/source/_static/css/installation_panel.css

```diff
@@ -16,7 +16,7 @@
     table-layout: fixed; /* make cells have even width */
 }
 
-.installation-panel-table tr, td{
+.installation-panel-table tr td{
```

Comment:
Change this line to `.installation-panel-table tr, .installation-panel-table td{`

Reply:
> Change this line to `.installation-panel-table tr, .installation-panel-table td{` and have a try again :)

thanks, this change works fine